omego/upgrade.py

```diff
@@ -218,7 +218,7 @@ def samecontents(a, b):
         else:
             if os.path.exists(target):
                 log.info('Deleting configuration file %s', target)
-                target.remove()
+                os.unlink(target)
```

Comment:
:+1: although this probably means none of the integration tests cover this codepaht at the moment?

Reply:
No, this is down to the weird `omero` behaviour of it sometimes creating a `config.xml` automatically even when no config commands are run.